tools/api-crawler/preprocessor.py

```
@@ -0,0 +1,1098 @@
+from sqlalchemy.engine import URL
+from sqlalchemy.orm import DeclarativeBase, Mapped, Session, mapped_column, relationship
+
+SERVICE_ID = "I1910"
+DEFAULT_INPUT_DIR = Path("json_raw")
+DEFAULT_SQLITE_PATH = Path("sqlite3/rag.sqlite3")
+PROJECT_ROOT = Path(__file__).resolve().parents[2]
+DEFAULT_BACKEND_ENV_PATH = PROJECT_ROOT / "backend" / ".env"
+DEFAULT_POSTGRES_HOST = "localhost"
+DEFAULT_POSTGRES_PORT = 5432
+DEFAULT_POSTGRES_USER = "postgres"
+DEFAULT_POSTGRES_PASSWORD = "root"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "DEFAULT_POSTGRES_PASSWORD"

(S105)